SECURITY.md

```diff
@@ -1,2 +1,2 @@
 - Please do not create GitHub issues to report security vulnerabilities. 
-- Instead, report them via http://eddiejaoude.io/contact.
+- Instead, report them via [Eddie Jaoude Contact Support Page](http://eddiejaoude.io/contact).
```

Comment:
For this error, we could use the following way to fix it.
```suggestion
- Instead, report them via <http://eddiejaoude.io/contact>.
```

Reply:
Works well too.